Rover/config.h

```diff
@@ -55,6 +55,9 @@
 # define MODE_FOLLOW_ENABLED AP_FOLLOW_ENABLED
 #endif
 
+#ifndef AP_LEAKDETECTOR_ENABLED
+#define AP_LEAKDETECTOR_ENABLED 1
+#endif
 
```

Comment:
```suggestion
```

Can't do this at the vehicle level.

If it is a problem on Plane and Copter then you have to "stub" things out in AP_LeakDetector.cpp if the vehicle doesn't want the feature.